src/app/rundown-overview/components/rundown-overview/rundown-overview.component.ts

```diff
@@ -41,7 +41,9 @@ export class RundownOverviewComponent implements OnInit, OnDestroy {
   }
 
   public openDeletionDialog(basicRundown: BasicRundown): void {
-    this.dialogService.openDeletionDialog('Rundown', basicRundown.name, () => this.deleteRundown(basicRundown.id))
+    this.dialogService.createConfirmDialog('Delete Rundown?', `Are you sure you want to delete the Rundown "${basicRundown.name}"?\n\nPlease note: This action is irreversible!`, 'Delete', () =>
```

Comment:
A note to all of us: We should also look into translation of text in the UI.

Reply:
We could look into how Angular handles it https://angular.io/guide/i18n-overview

Reply:
I have created a story for it https://tv2cms.atlassian.net/browse/SOF-1640